source/drivers/driver-compatibility-reference.txt

```diff
@@ -718,6 +786,14 @@ Language Compatibility
      - PHP 7.0
      - HHVM 3.9
 
+   * - mongodb-1.2
+     -
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
```

Comment:
@jmikola/@derickr Which driver versions support PHP 7.1 (so I can put the appropriate checkmarks)? I assume the same as support PHP7, but best to be sure :)